### DISCUSSION TITLE
What version of "column" does ubuntu-20.04 use?

### DISCUSSION THREAD

**Discussion by bewuethr** [actions/virtual-environments#2936] [Installed software]
I was wondering which version of the `column` tool the ubuntu-20.04 environment includes. My personal Ubuntu machine (using 20.10) has `column` from the util-linux package, and since that package is marked "essential", I was assuming it's installed without being mentioned in the "installed apt packages".

However, it seems that the `column` that *is* installed is a more minimal version; trying to use long options with `--` resulted in just an error message with the usage hint:

```txt
column: invalid option -- '-'
usage: column [-txne] [-c columns] [-s sep] [file ...]
```

So, what version of `column` is included in the ubuntu-20.04 environment?

**Comment by miketimofeev** [comment]
Hi @bewuethr!
The version that is included in bsdmainutils http://manpages.ubuntu.com/manpages/focal/man1/column.1.html

**Reply by bewuethr** [reply]
Thanks, @miketimofeev! Is it an oversight then that bsdmainutils isn't listed under "Installed apt packages", or is it part of 20.04 "out of the box"?

**Reply by miketimofeev** [reply]
yes, looks like it's a part of the distribution because I can't find any entry about it in the image generation log

**Reply by bewuethr** [reply]
And (last question, promise) why isn't it the `column` from util-linux then? Is that a 20.04 vs. 20.10 difference?

**Reply by miketimofeev** [reply]
Indeed — judging by this page column in 20.10 is from bsdextrautils http://manpages.ubuntu.com/manpages/groovy/man1/column.1.html

**Reply by bewuethr** [reply]
Makes sense, thanks! 🎉

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
